packages/applications-service-api/api/openapi.yaml

```diff
@@ -301,6 +301,42 @@ paths:
         '404':
           description: No documents found
 
+  '/api/v3/documents/{caseReference}':
+    get:
+      summary: Returns project documents
+      description: Returns project documents
+      tags:
+        - Documents
+      parameters:
+        - in: path
+          name: caseReference
+          example: 'TR040009'
+          required: true
+          schema:
+            type: string
+          description: Case reference
+        - in: query
+          name: type
+          description: Type of document
+          schema:
+            type: string
+            enum: [
+              'RULE 6 LETTER',
+              'RULE 8 LETTER',
+              'EXAMINATION LIBRARY',
+              'DCO DECISION LETTER (SOS)(APPROVE)',
+              'DCO DECISION LETTER (SOS)(REFUSE)'
+            ]
+      responses:
+        '200':
+          description: Returns list of documents
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/DocumentListWrapper'
```

Comment:
`DocumentListWrapper` has all the pagination and filtering stuff, and this endpoint is returning just a single document so i think `DocumentV3` is what we want here